Refer A Friend Incentive Copy (#2295)
DoSomething/phoenix-next: docs/development/features/referral-pages.md

- **Alpha** - The user who is sharing the URL with friends, prompting the friends to register
- **Beta** - The friends that the alpha is sharing the URL with

Users will see a Referral Page Banner after signing up for any campaign configured for this feature, which refers them to their Alpha Referral Page.

> We offer two versions of the RAF copy across all referral pages. One - incentive specific, alluding to a \$10 gift card reward for the Alpha & Beta, and the second, an incentive-less version.
> 
> The `DS_ENABLE_REFER_FRIENDS_INCENTIVE` ENV variable toggles between the two. If set to `true`, the incentive specific copy will display.

Reviewer: The first paragraph here made me think this is something like A/B testing, until I got to 2nd paragraph. What do you think about this edit?
```suggestion
When the `DS_ENABLE_REFER_FRIENDS_INCENTIVE` ENV variable is set to `true`, incentive-specific RAF copy is displayed across all referral pages, alluding to a \$10 gift card reward for the Alpha & Beta. Otherwise incentive-less RAF copy is displayed.

```

Author: Love this alternative!
